site/content/docs/groups/_index.md

```diff
@@ -40,11 +40,63 @@ The screenshot below gives a quick overview of the groups' key features.
     5. CI/CD triggers
```

Comment:
@guolau I would remove CI/CD triggers here as the old "Command Line Triggers" feature has been superseded by the CLI, and the CLI's `npx checkly trigger` command does not have a way to trigger groups per se.